### DISCUSSION TITLE
Docusaurus doesn't accept plugin devServer configs anymore

### DISCUSSION THREAD

**Discussion by Anthony-Gaudino** [facebook/docusaurus#6100] [General] [answered]
I'm using Docusaurus 2.0.0-beta.9 along with [docusaurus-plugin-openapi ](https://github.com/cloud-annotations/docusaurus-plugin-openapi) and I have the issue described here https://github.com/cloud-annotations/docusaurus-plugin-openapi/issues/78 where the plugin maintainer says that Docusaurus removed support for `devServer configs`.

I need to pass the proxy setting to the plugin so that it will allow me to send requests to my particular server URL and port.

Is there a workaround for this?

**Comment by slorber** [comment, marked as answer]
Thanks, I'll fix this in https://github.com/facebook/docusaurus/pull/6107
